libs/model/test/email/digest-email-lifecycle.spec.ts

```diff
@@ -1,19 +1,24 @@
 import { ExternalServiceUserIds, dispose, query } from '@hicommonwealth/core';
 import { models } from '@hicommonwealth/model';
-import { Community } from '@hicommonwealth/schemas';
+import { Community, User } from '@hicommonwealth/schemas';
+
 import { expect } from 'chai';
```

Comment:
we are trying to use expect from vitest, to eventually remove chai from the project